### PR TITLE
Oleg's project(Internationalization)

### PR DESCRIPTION
**Pull request by OlegAndoniy**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -12,7 +12,7 @@
         <div class="top">
             <div class="budget">
                 <div class="budget__title">
-                    Available Budget in <span class="budget__title--month">%Month%</span>:
+                    Available Budget in <span id="budget_title_month" class="budget__title--month">%Month%</span>:
```
Comment (egrig) [inline index.html]: @OlegAndoniy Why did you add an id tag here?

File: index.html [hunk 2]
```diff
@@ -44,15 +44,32 @@
                         <option value="inc" selected>+</option>
                         <option value="exp">-</option>
                     </select>
-                    <input type="text" class="add__description" placeholder="Add description">
-                    <input type="number" class="add__value" placeholder="Value">
+                    <input type="text"  name="description" class="add__description" placeholder="Add description">
+                    <input type="number" name="value" class="add__value" placeholder="Value">
+
```
Comment (egrig) [inline index.html]: Why did you add name parameters here? To find the HTML elements better?

File: index.html [hunk 3]
```diff
@@ -44,15 +44,32 @@
...
+                    
+
+
                 </div>
             </div>
             
             <div class="container clearfix">
                 <div class="income">
-                    <h2 class="income__title">Income</h2>
+                    <h2 class="icome__title">Income</h2>
```
Comment (egrig) [inline index.html]: Spelling error?

File: app.js [hunk 4]
```diff
@@ -156,248 +184,491 @@ var budgetController = (function() {
...
+        expenseBottom: '.expenses__title',
+        budgetText: '.budget__title',
+        german: '#german',
+        ukrainian: '#ukrainian',
+        english: '#english',
+        russian: '#russian',
+        chinese:'#chinese',
+        italian:'#italian',
+        french:'#french',
+        spanish:'#spanish'
```
Comment (egrig) [inline app.js]: In your opinion, do you think all these DOM strings should be in the UI controler or here?

File: app.js [hunk 5]
```diff
@@ -156,248 +184,491 @@ var budgetController = (function() {
...
+       },
+
+  browserLang: function()
+  {
+        var x;
+        x=navigator.language;
+    //     y=navigator.languages;
+    //    console.log(y);
+    //    console.log(x);
+       switch(x){
```
Comment (egrig) [inline app.js]: Nice use of the switch statements!

File: index.html (hunk 2 above)
Comment (OlegAndoniy) [reply index.html]: to use method getElementsByName

File: index.html (hunk 1 above)
Comment (OlegAndoniy) [reply index.html]: Forgot delete it

File: index.html (hunk 3 above)
Comment (OlegAndoniy) [reply index.html]: Yes

File: app.js (hunk 4 above)
Comment (OlegAndoniy) [reply app.js]: I think in language controller

File: app.js (hunk 5 above)
Comment (OlegAndoniy) [reply app.js]: Thanks
